src/pages/test-report/components/toolbar/toolbar.ts

```diff
@@ -55,15 +62,30 @@ export class ToolbarComponent {
       this.subscription.unsubscribe();
     }
   }
+  toggleremoveFaultMode(): void {
+    if (this.isDangerousMode) {
```

Comment:
Toggling dangerous or serious mode off when you turn on remove mode is not the desired behaviour because there are scenarios when we have to remove a serious fault for example and then we need to have both of them toggled on regardless of order.